guide/src/class.md

```diff
@@ -828,7 +830,12 @@ impl MyClass {
 
 ## Classes as function arguments
 
-Free functions defined using `#[pyfunction]` interact with classes through the same mechanisms as the self parameters of instance methods, i.e. they can take Python-bound references, Python-bound reference wrappers or Python-independent references:
+Class objects can be used as arguments to `#[pyfunction]`s and `#[pymethods]` in the same way as the self parameters of instance methods, i.e. they can be passed as:
+- `Py<T>` or `Bound<'py, T>` smart pointers to the class Python object,
+- `&T` or `&mut T` references to the Rust data contained in the Python object, or
+- `PyRef<T>` and `PyRefMut<T>` reference wrappers.
```

Comment:
Should we also mention `PyClassGuard` here? Or do we wait with that until we deprecate `PyRef` eventually?

Reply:
I was wondering about it, I think it would be good to introduce `PyClassGuard` as part of 0.27 if we can figure out the way to sequence it in! I would prefer defer from this PR for the moment though.